devices/oneplus-fajita/kernel/default.nix

```diff
@@ -6,14 +6,14 @@
 }:
 
 mobile-nixos.kernel-builder rec {
-  version = "5.12.15";
-  modDirVersion = "5.12.15";
+  version = "5.14.0-rc6";
+  modDirVersion = "5.14.0-rc6";
   configfile = ./config.aarch64;
   src = fetchFromGitLab {
     owner = "sdm845-mainline";
     repo = "linux";
-    rev = "089737886ff3b793ccece369b88ee0ae4b39b9f5";
-    sha256 = "sha256-TM+pdweCX/VhKnjusvBUdpgPdQibywvSnsYAWI0mL8w=";
+    rev = "94a1a791d2559c8224a6dbd58a0a7e59d7c85712";
```

Comment:
I had an issue on later kernels where the display would fade to black after the splash screen. I was unable to reach either Phosh or the recovery menu. I still need to look into this more, but for now, this works.